### PR TITLE
Address suggestions

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #25486 

Resolves suggestions through docs/framework/data folder alphabetically (except for csharp and architecture).

[Hide whitespace changes in diff.](https://github.com/dotnet/docs/pull/25624/files?diff=unified&w=1)

[Sample preview link](https://review.docs.microsoft.com/en-us/dotnet/framework/data/adonet/ef/language-reference/or-entity-sql?branch=pr-en-us-25624).

### CODE REVIEW

File: docs/framework/data/adonet/sql/linq/standard-query-operator-translation.md [hunk 1]
```diff
@@ -142,12 +142,22 @@ The following helper functions that are used by the Visual Basic compiler are tr
...
+- ToDecimal
+- ToDouble
+- ToInteger
+- ToUInteger
+- ToLong
+- ToULong
+- ToShort
+- ToUShort
+- ToSingle
+- ToString
```
Comment (IEvangelist) [inline docs/framework/data/adonet/sql/linq/standard-query-operator-translation.md:160]:
```suggestion
- `ToBoolean`
- `ToSByte`
- `ToByte`
- `ToChar`
- `ToCharArrayRankOne`
- `ToDate`
- `ToDecimal`
- `ToDouble`
- `ToInteger`
- `ToUInteger`
- `ToLong`
- `ToULong`
- `ToShort`
- `ToUShort`
- `ToSingle`
- `ToString`
```

**Review by IEvangelist — APPROVED** [review]
Just one nit... otherwise looks great! :shipit:

**Review by BillWagner — APPROVED** [review]
This LGTM. I don't have anything additional to the comments made by @IEvangelist

**Comment by gewarren** [comment]
Thanks for the reviews!

**Pull request merged**